src/Nirum/Targets/Docs.hs

```diff
@@ -161,6 +164,16 @@ $doctype 5
     documentSortKey ("", _) = (False, 0, "")
     documentSortKey (fp@(fp1 : _), _) =
         (isUpper fp1, length (filter (== pathSeparator) fp), fp)
+    hljsCss :: T.Text
+    hljsCss = T.concat
+        [ "https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0"
+        , "/styles/github.min.css"
+        ]
+    hljsJs :: T.Text
+    hljsJs = T.concat
+        [ "https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/"
```

Comment:
As `hljsJs` and `hljsCss` share the common prefix, how about extract out the common prefix?  It would be also helpful when we need to upgrade the version of the highlight.js library.